src/hotspot/share/oops/methodCounters.cpp

```diff
@@ -79,9 +76,6 @@ void MethodCounters::clear_counters() {
 
 void MethodCounters::metaspace_pointers_do(MetaspaceClosure* it) {
```

Comment:
MethodCounters::metaspace_pointers_do can be removed altogether (also need to remove the declaration in methodCounter.hpp).

Reply:
Done.